Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -238,8 +238,15 @@ open class AVFoundationPlayback: Playback {
     open override func seek(_ timeInterval: TimeInterval) {
         let time = CMTimeMakeWithSeconds(timeInterval, Int32(NSEC_PER_SEC))
 
-        player?.currentItem?.seek(to: time)
         trigger(.seek)
+        trigger(.willSeek)
```

Comment:
Why did you keep the ``.seek`` event here?

Reply:
We will keep the `seek` event for now and update the docs that it will be deprecated. If we remove it now we can break all of our users that listen to this event.